site/content/en/docs/Reference/Commands/start.md

```diff
@@ -16,6 +16,7 @@ minikube start [flags]
 ### Options
 
 ```
+--addons                            Enable addons. see `minikube addons list` if you want to check
```

Comment:
Let's change this description to "Enables addons. See `minikube addons list` for a list of valid addon names."

Reply:
Thanks to you, the help message became easier to understand!
I changed.